wolox-books/src/views/Register.vue

```diff
@@ -0,0 +1,129 @@
+<template lang="pug">
+.container
+  form.form-container(@submit.prevent="onSubmit" novalidate)
+    img.wolox-image(src="../assets/wolox.png" alt="Wolox")
+    .input-container
+      label.input-label
+        | First name
+      input.input-field(type="text" v-model="firstName")
+    .input-container
+      label.input-label
+        | Last name
+      input.input-field(type="text" v-model="lastName")
+    .input-container
+      label.input-label
+        | Email
+      input.input-field(type="email" v-model="email")
+    .input-container
+      label.input-label
+        | Password
+      input.input-field(type="password" v-model="password")
+    .input-container
+      label.input-label
+        | Confirm password
+      input.input-field(type="password" v-model="confirmPassword")
+    button.green-button.submit-button(type="submit")
+      | Sign up
+    button.green-button.login-button
+      | Log in
+</template>
+
+<script>
+export default {
+  name: 'register',
+  data () {
+    return {
+      firstName: null,
+      lastName: null,
+      email: null,
+      password: null,
+      confirmPassword: null,
+      locale: 'en'
+    }
+  },
+  methods: {
+    onSubmit() {
+      const productReview = {
+        firstName: this.firstName,
+        lastName: this.lastName,
+        email: this.email,
+        password: this.password,
+        confirmPassword: this.confirmPassword
+      }
+      console.log(productReview)
+      this.firstName = null
+      this.lastName = null
+      this.email = null
+      this.password = null
+      this.confirmPassword = null
+    }
+  }
+}
+</script>
+
+<style scoped lang="scss">
+@import '../scss/aplication.scss';
```

Comment:
Don't import the application, import only the variables needed. In this case, `colors`